schema.sql

```diff
@@ -34,4 +34,26 @@ ALTER TABLE animals
 ADD CONSTRAINT fk_species FOREIGN KEY (species_id) REFERENCES species(id);
 
 ALTER TABLE animals
-ADD CONSTRAINT fk_owner FOREIGN KEY (owner_id) REFERENCES owners(id);
\ No newline at end of file
+ADD CONSTRAINT fk_owner FOREIGN KEY (owner_id) REFERENCES owners(id);
+
+CREATE TABLE vets (
+  id INTEGER GENERATED ALWAYS AS IDENTITY,
+  name VARCHAR(200),
+  age INTEGER,
+  date_of_graduation DATE,
+  PRIMARY KEY (id)
+);
+
+CREATE TABLE specializations (
+    vet_id INTEGER REFERENCES vets(id),
+    species_id INTEGER REFERENCES species(id),
+    vet_name varchar(300),
+    species_name varchar(200),
+    PRIMARY KEY (vet_id, species_id)
+);
+
+CREATE TABLE visits (
+  vets_name VARCHAR(300),
+  animal_name VARCHAR(200),
+  visit_date DATE
```

Comment:
- [ ] Great job implementing the **join table** `visits` :clap: However, it's important to use the primary key to establish relationships in a many-to-many relationship, rather than relying on names. The `vets_name` and `animal_name` are not unique and that makes it difficult to maintain, and query data.
  Therefore I suggest you use the **primary key** of the `vets` and `animal` tables instead of the name.

Example:
```sql
CREATE TABLE visits (
   animal_id int,
   vet_id int,
   visit_date date
);
```